tools/test.py

```diff
@@ -53,6 +53,11 @@ def parse_args():
         type=float,
         default=0.3,
         help='score threshold (default: 0.3)')
+    parser.add_argument(
+        '--show-colors',
```

Comment:
Please provide a simple example to illustrate the usage.

Reply:
--show-colors can be a single color or a color file.
```
# single color input
python tools/test.py \
    configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py \
    checkpoints/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth \
    --show \
    --show-colors green
```

```
# single color file
python tools/test.py \
    configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py \
    checkpoints/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth \
    --show \
    --show-colors colors.txt
```
[colors.txt](https://github.com/open-mmlab/mmdetection/files/7702066/colors.txt)

Reply:
It is not a good way to load colors from txt file